mucommander-core/src/main/java/com/mucommander/ui/terminal/TerminalIntegration.java

```diff
@@ -62,6 +62,7 @@ public class TerminalIntegration {
      * When divider is close to max by this value we conclude it is maximised.
      */
     private static final float TREAT_AS_MAXIMIZED = 0.3f;
+    private static final String CHANGE_DIR_FORMAT = "cd \"%s\"" + System.lineSeparator();
```

Comment:
the original command had space before deliberately to avoid storing the command in (bash) history.

```
" cd \"" + newCwd + "\""
```

Reply:
ack, thanks @pskowronek - changed